### ISSUE TITLE
make migrations manual

### ISSUE THREAD

**Issue by jd2rogers2** [Nonprofit-Exchange-Hub/web-app#64]
ACs:
- https://github.com/Nonprofit-Exchange-Hub/web-app/blob/c8a1d6b2c6e3894b243e4f926444e81155eb38e5/server/src/database-connection.service.ts#L15 should be set to false so that we don't automatically clear previous tables every time we start a new connection
- use resources below to set up a new workflow for when we update entity schemas
  - keep in mind that this new workflow will also be utilized in deployment build scripts
- It should be enough to have the `typeorm`-generated sql transactions there
- (DONE) provide some seed data (arguably separate PR, let's chat if this ends up being difficult)
  - https://typeorm.io/#/insert-query-builder
- update set up instructions in readme to reflect new migration step
  - and maybe some how to documentation for day to day use of these as well

### Acceptance criteria
- we should have manual typeorm migrations in place that we can run from a script. similar to the `seed:drop` script
- Optional: add the script to a process on the docker-compose for development, so that the migration runs on `docker-compose up`
- We can see a clear flow of creating a new database, running the migrations and seeding the db

resources:
https://docs.nestjs.com/techniques/database#migrations
https://typeorm.io/#/migrations
https://typeorm.io/#/using-cli

Blocks: https://github.com/Nonprofit-Exchange-Hub/web-app/issues/308
